README.md

```diff
@@ -1 +1,34 @@
-# Mail-Service
\ No newline at end of file
+# Mail-Service
+
+Welcome to our Mail- Service !
+Here we manage email templates, and send out your custom emails for now or for later. You can write your own template by editing the HTML code, save, create and delete the templates. You can choose to send a new E-mail, with all the usual information (CC, BCC, subject, to). As for the body you can either type in plain text, or choose a template, and then also choose to edit the template HTML before you send to customize further. You may choose to send out the email right away, or schedule it for a future time. Later you can see stats about the emails sent, the future emails, and the templates.
+Before you start:
```

Comment:
this file should be according to the md file best practices, please [read this](https://www.freecodecamp.org/news/how-to-write-a-good-readme-file/) and follow [this template](https://github.com/nuxeo/nuxeo/blob/master/README.md).